crates/emitter/src/emitter.rs

```diff
@@ -1368,6 +1357,10 @@ impl InstructionWriter {
 
     // @@@@ END METHODS @@@@
 
+    pub fn get_atom_gcthing_index(&mut self, atom: SourceAtomSetIndex) -> GCThingIndex {
+        self.gcthings.push_atom(atom)
```

Comment:
This is the duplication I'm wondering about.

Edit: I see you noticed now too :)

Reply:
thanks!
fixed